### PR TITLE
Pull upstream only args into a conditional.

### PR DESCRIPTION
**Pull request by myersCody**

### CODE REVIEW

File: qpc_tools/cli/install.py [hunk 1]
```diff
@@ -39,16 +40,19 @@ def __init__(self, subparsers):
...
+        if not DOWNSTREAM:
+            # Upstream only args
+            self.parser.add_argument('--offline', dest='install_offline',
+                                     action='store_true',
+                                     help=_(messages.CLI_INSTALL_OFFLINE_HELP),
+                                     required=False)
+            self.parser.add_argument('--offline-files', dest='offline_files',
+                                     help=_(messages.CLI_INSTALL_OFFLINE_FILES_HELP),
+                                     required=False)
+            self.parser.add_argument('--version', dest='cli_version',
```
Comment (kholdaway) [inline qpc_tools/cli/install.py]: Let's pull this out since it applies to both upstream and downstream.

File: qpc_tools/server/install.py [hunk 2]
```diff
@@ -48,17 +48,19 @@ def __init__(self, subparsers):
...
+        else:
+            # Upstream only args
+            self.parser.add_argument('--offline', dest='install_offline',
+                                     action='store_true',
+                                     help=_(messages.SERVER_INSTALL_OFFLINE_HELP),
+                                     required=False)
+            self.parser.add_argument('--offline-files', dest='offline_files',
+                                     help=_(messages.SERVER_INSTALL_OFFLINE_FILES_HELP),
+                                     required=False)
+            self.parser.add_argument('--version', dest='server_version',
```
Comment (kholdaway) [inline qpc_tools/server/install.py]: Let's pull this out since it applies to both upstream and downstream.

**Review by kholdaway — CHANGES_REQUESTED** [review]
Version is applicable to both up and downstream.

**Review by abaiken — APPROVED** [review]
Nice work!

**Review by kholdaway — APPROVED** [review]

**Pull request merged**